.gitignore

```diff
@@ -6,7 +6,8 @@
 /out-tsc
 
 # dependencies
-/node_modules
+node_modules
+api/node_modules/
```

Comment:
we shouldn't need this as the line above should capture all 'node_modules' folders

Reply:
maybe change the pattern to node_modules/